ethereum-rpc/src/constructors/express/routes/api/v1/uniswap.js

```diff
@@ -405,98 +254,31 @@ module.exports = ({
       try {
         const errors = validationResult(req);
         if (!errors.isEmpty()) {
           return res.status(400).json({ errors: errors.array() });
         }
-        const { address = "*" } = req.params;
-        const { amount = "", tokenFrom = "", tokenTo = "", network = "" } = { ...req.query };
+        const { tokenFrom = "", tokenTo = "", network = "", tokenFromAddress = "", tokenToAddress = "" } = { ...req.query };
         if (tokenFrom === tokenTo) {
           res.status(500).json({ error: "tokenFrom and tokenTo cannot be same" })
         }
-        const tokenTypeFrom = tokenFrom.toString().toUpperCase();
-        const tokenTypeTo = tokenTo.toString().toUpperCase();
-        const networkType = network.toString().toUpperCase();
-
-        let chainIds = {}
-        switch (networkType) {
-          case 'MAINNET':
-            chainIds = ChainId.MAINNET;
-            break;
-          case 'RINKEBY':
-            chainIds = ChainId.RINKEBY;
-            break;
-        }
-        let tokenAddressDai, tokenAddressUniswap, tokenAddressMaker, tokenAddressWrappedEther
-        if (networkType === "MAINNET") {
-          tokenAddressDai = '0x6B175474E89094C44Da98b954EedeAC495271d0F'
-          tokenAddressUniswap = '0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984';
-          tokenAddressMaker = '0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2';
-          tokenAddressRouter = '0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D'
-          tokenAddressWrappedEther = '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
-        }
-        else if (networkType === "RINKEBY") {
-          tokenAddressDai = '0xc7ad46e0b8a400bb3c915120d284aafba8fc4735'
-          tokenAddressUniswap = '0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984';
-          tokenAddressMaker = '0xF9bA5210F91D0474bd1e1DcDAeC4C58E359AaD85';
-          tokenAddressRouter = '0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D'
-          tokenAddressWrappedEther = '0xc778417E063141139Fce010982780140Aa0cD5Ab';
-        }
-        const DAI = await Fetcher.fetchTokenData(chainIds, tokenAddressDai);
-        const UNISWAP = await Fetcher.fetchTokenData(chainIds, tokenAddressUniswap);
-        const MAKER = await Fetcher.fetchTokenData(chainIds, tokenAddressMaker);
-        const WETHS = await Fetcher.fetchTokenData(chainIds, tokenAddressWrappedEther);
-
-        let tokenFromInstance = {};
-        switch (tokenTypeFrom) {
-          case 'DAI':
-            tokenFromInstance = DAI;
-            break;
-          case 'UNI':
-            tokenFromInstance = UNISWAP;
-            break;
-          case 'MKR':
-            tokenFromInstance = MAKER
-            break;
-          case 'WETH':
-            tokenFromInstance = WETHS
-            break;
-          case 'ETH':
-            tokenFromInstance = WETHS
-            break;
+        if (tokenFromAddress === tokenToAddress) {
+          res.status(500).json({ error: "tokenFromAddress and tokenToAddress cannot be same" })
         }
```

Comment:
again, `return` -- however, the return will be handled by the express-validator if we handle it there

Reply:
its because , the same set of code was used in all routes .... will update it